Gems/Atom/Feature/Common/Code/Source/ScreenSpace/DeferredFogPass.cpp

```diff
@@ -180,6 +194,11 @@ namespace AZ
 
         bool DeferredFogPass::IsEnabled() const 
         {
+            if (!r_enableFog)
```

Comment:
not a bad idea, I could this being disabled in the RV scene and then enabled in the level.

Reply:
Yes, it works at runtime.

But I won't suggest to use this for game feature since you want to use this as part of quality settings.
If you change it at runtime, how about the devices which always disable the fog. 

And for the use case Alex mentioned, you don't need to use this setting. user just need to make sure not add fog component in the RV scene and add the fog component in the level.

Also, the fog component has enable/disable setting which can the work too.